db/glue.c

```diff
@@ -504,11 +504,9 @@ int trans_abort_shadow(void **trans, int *bdberr)
     return rc;
 }
 
-static int trans_commit_seqnum_int(void *bdb_handle, struct dbenv *dbenv,
-                                   struct ireq *iq, void *trans,
-                                   db_seqnum_type *seqnum, int logical,
-                                   void *blkseq, int blklen, void *blkkey,
-                                   int blkkeylen)
+static int trans_commit_seqnum_int(void *bdb_handle, struct dbenv *dbenv, struct ireq *iq, void *trans,
+                                   db_seqnum_type *seqnum, int logical, uint32_t *logbytes, void *blkseq, int blklen,
```

Comment:
Doesn't look like `logbytes` is used in this function.

Reply:
Oops- that was a mistake .. thanks Rivers!